### ISSUE TITLE
Startup time drastically increased

### ISSUE THREAD

**Issue by BAPACop** [araten10/EdgewarePlusPlus#84]
In version 11 startup time, between clicking the .pyw file to launch and the splash screen appearing, was about 2-3 seconds. I skipped version 12-12.75, but in version 13b the startup time is two and a half hours, with disk usage for the drive that the files are on at 100% for the entire time.

**Comment by LewdDevelopment** [comment]
That is strange, I haven't encountered any performance problems like this. Does this happen consistently? Does it happen with any particular settings or even if you set all the settings to the default ones? Are you sure this couldn't be caused by something else on your computer?

**Comment by BAPACop** [comment]
100% of the time at launch, even with default config. I even uninstalled Python and reran the setup just to be sure. I watched with resource monitor and it is accessing every file in the resource folder as it starts up,  It doesn't seem to be a massive operation, but it adds up at large file counts. Taking files out of the resource folder speeds the launch up.

**Comment by LewdDevelopment** [comment]
If you change the last line of `edgeware/src/pack/load.py` which looks like this
```python
    return [Media(dir / file, media_moods.get(file, None)) for file in os.listdir(dir) if is_valid(dir / file)] if dir.is_dir() else []
```
into this
```python
    return [Media(dir / file, media_moods.get(file, None)) for file in os.listdir(dir)] if dir.is_dir() else []
```
does that help?

**Comment by BAPACop** [comment]
Yes, startup goes back to being just a few seconds again with that change.
